jobs/JGLOBAL_FORECAST

```diff
@@ -17,15 +17,23 @@ rCDUMP=${CDUMP}
 [[ ${CDUMP} = "gfs" ]] && export rCDUMP="gdas"
 
 # Forecast length for GFS forecast
-if [ ${CDUMP} = "gfs" ]; then
+case ${CDUMP} in
+  gfs | gefs)
+    # shellcheck disable=SC2153
     export FHMAX=${FHMAX_GFS}
+    # shellcheck disable=SC2153
     export FHOUT=${FHOUT_GFS}
     export FHMAX_HF=${FHMAX_HF_GFS}
     export FHOUT_HF=${FHOUT_HF_GFS}
-else
+    ;;
+  gdas)
     export FHMAX_HF=0
     export FHOUT_HF=0
-fi
+    ;;
+  *)
+    echo "FATAL ERROR: Unknown CDUMP '${CDUMP}'"
```

Comment:
Should we use `RUN` instead of `CDUMP` here and in the case statement?

Reply:
Can, just makes the cases longer

Reply:
Done. Doesn't really make the case statements longer once I thought about it.